README.md

```diff
@@ -108,6 +110,59 @@ type RouteClient struct {
 func GetRoute(destination string) *Route { ... }
 ```
 
+## The case of an initialism should be consistent
+
+In code, the case of an initialism should be consistent, i.e. either all
+uppercase or lowercase. For example, "TCP" should be written as "TCP" or "tcp",
+e.g. `ProtocolTCP`, `tcpHeader`.
+
+In comments and documentation, initialisms should be all uppercase, e.g.
+"Protocol could be UDP, TCP, or SCTP".
+
+```go
+// Bad example
+// TcpHeader contains the tcp specific header.
+type TcpHeader struct { ... }
+
+func (p *Packet) SetTcpHeader(Tcp *TcpHeader) { ... }
+
+// Good example
+// TCPHeader contains the TCP specific header.
+type TCPHeader struct { ... }
+
+func (p *Packet) SetTCPHeader(tcp *TCPHeader) { ... }
+```
+
+## Write proper nouns formally
+
+In comments and documentation, use **upper camel case** for K8s resource and CRD
+kind names, e.g. "Pod", "Node", "Service", "NetworkPolicy", "TrafficControl".
+For component names, **capitalize the first letters** if a component name
```

Comment:
Should we start a separate paragraph after the resource / CRD examples?

Reply:
done